copyspecial.py

```diff
@@ -6,36 +6,86 @@
 # Google's Python Class
 # http://code.google.com/edu/languages/google-python-class/
 
+
```

Comment:
The first line of docstrings is the summary line. It should follow the triple quotation, on the same line, and should end with a period.

Otherwise, great job on your module docstring!

Reply:
So is this summary supposed to be very short then? and i have updated it with an attempt at a short summary.